README.md

```diff
@@ -55,7 +55,7 @@ A full example leveraging other community modules is contained in the [test/fixt
 2. Define the ALB module using the following in your [main.tf](test/fixtures/main.tf):
 ```
 module "alb" {
-  source              = "github.com/terraform-community-modules/tf_aws_alb//alb"
+  source              = "github.com/terraform-community-modules/tf_aws_alb/alb"
```

Comment:
@khalidharun if you revert this bit of the PR I'll be happy to merge it in. Thanks for the link update!